vscode-dbt/snippets/snippets_sql.json

```diff
@@ -133,6 +133,17 @@
         ],
         "description": "Env Var"
     },
+    "Model Description": {
```

Comment:
I would maybe rename this to `"Model General Comment"` and therefore the prefix should be something like `model_general_comment`. 

I'm afraid that people start thinking this is equivalent to the description of the model which resides in the `schema.yml`

What do you think?